bunq/sdk/json/anchored_object_model_adapter.py

```diff
@@ -0,0 +1,70 @@
+from bunq import AnchoredObjectInterface
+from bunq.sdk.exception.bunq_exception import BunqException
+from bunq.sdk.json import converter
+from bunq.sdk.model.generated import endpoint
+from bunq.sdk.model.generated import object_
+
+
+class AnchoredObjectModelAdapter(converter.JsonAdapter):
+    _ERROR_MODEL_NOT_FOUND = '{} is not in endpoint nor object.'
+
+    __STRING_FORMAT_UNDERSCORE = '_'
```

Comment:
why is one constant private and the other protected?

Reply:
Make both private.